tests/test_db.py

```diff
@@ -288,4 +298,293 @@ def test_form_sorting(seed_dynamic_data, _db):
     assert result_section.forms[1].form_id == form1.form_id
     assert result_section.forms[2].form_id == formX.form_id
     assert result_section.forms[3].form_id == form2.form_id
-    assert result_section.forms[3].section_index == 3
+    assert result_section.forms[3].section_index == 4
+
+
+section_id = uuid4()
+
+
+@pytest.mark.seed_config(
```

Comment:
Can we rename the file into `test_db_seed` or similar thing when we look the file name it seems like going to write db level tests